packages/contracts/src/DatabaseResolver.sol

```diff
@@ -113,53 +114,16 @@ contract DatabaseResolver is
 
     /**
      * Forwards the registering of a domain to the L2 contracts
-     * @param -name DNS-encoded name to be registered.
-     * @param -owner Owner of the domain
-     * @param -duration duration The duration in seconds of the registration.
-     * @param -secret The secret to be used for the registration based on commit/reveal
-     * @param -resolver The address of the resolver to set for this name.
-     * @param -data Multicallable data bytes for setting records in the associated resolver upon reigstration.
-     * @param -reverseRecord Whether this name is the primary name
-     * @param -fuses The fuses to set for this name.
-     * @param -extraData any encoded additional data
+     * @param -request The registration request details containing name, owner, duration, resolver, etc
      */
-    function register(
-        bytes calldata, /* name */
-        address, /* owner */
-        uint256, /* duration */
-        bytes32, /* secret */
-        address, /* resolver */
-        bytes[] calldata, /* data */
-        bool, /* reverseRecord */
-        uint16, /* fuses */
-        bytes memory /* extraData */
-    )
+    function register(RegisterRequest calldata /* request */ )
         external
         payable
         override
     {
         _offChainStorage();
     }
 
-    //////// OFFCHAIN STORAGE TRANSFER DOMAIN ////////
-
-    /**
-     * Transfer a domain to a new owner
-     * @param -node The DNS-encoded name to resolve.
-     * @param -owner The address of the new owner
-     */
-    function transfer(bytes32, /* node */ address /* owner */ ) external view {
```

Comment:
The domain cannot be transferred anymore?

Reply:
the transfer function should now be called through the `getDeferralHandler` and than redirected to the Gateway, therefore, this function isn't need anymore on the DatabaseResolver. 

however, we could have this function just for ABI usage and for direct usage. I'll remove it for now